generate_homepage_xml.py

```diff
@@ -46,10 +46,8 @@ def page_data_handling(data_list: list, get_unique_per_dev=False):
     xml_gen = GenerateXML()
     elastic_search = ElasticSearchClient()
     dev_urls = [
-        ["https://lists.linuxfoundation.org/pipermail/bitcoin-dev/",
-         "https://gnusha.org/pi/bitcoindev/",
+        ["https://gnusha.org/pi/bitcoindev/",
          "https://mailing-list.bitcoindevs.xyz/bitcoindev/"],

```

Comment:
here we have an array inside an array?

Reply:
Same as above - https://github.com/bitcoinsearch/summarizer/pull/66#discussion_r1838331497